store.go

```diff
@@ -738,39 +745,81 @@ func (s *store) GraphDriver() (drivers.Driver, error) {
 	return s.getGraphDriver()
 }
 
-// LayerStore obtains and returns a handle to the writeable layer store object
+// RWLayerStore obtains and returns a handle to the writeable layer store object
```

Comment:
That's an API break which would bump us to v2.0.0 (forcing us into the version prefixing of go modules). I think we should keep `LayerStore` around and make it call `RWLayerStore`.

Reply:
Yes, that makes sense. I change it as suggested.